tests/python/test_matmul.py

```diff
@@ -201,3 +201,24 @@ def fusion_func(fd: FusionDefinition) -> None:
         ]
         outputs, _ = self.exec_nvfuser(fusion_func, inputs)
         assert outputs[0].ndim == 3
+
+    def test_matmul_stride(self):
+        b, m, n, k = 3, 2, 5, 4
+        inputs = [
+            torch.randn(b, b, m, k, device="cuda", dtype=torch.float16),
```

Comment:
It's valid but weird to have two batch dimensions in input `a`. Would you mind making this more realistic?

Reply:
Switched it to be of sdpa pattern.